src/main/java/com/aitusoftware/babl/monitoring/MappedSessionStatistics.java

```diff
@@ -34,7 +34,8 @@ public final class MappedSessionStatistics extends SessionStatistics implements
     private static final int FRAMES_DECODED_OFFSET = SEND_BUFFERED_BYTES_OFFSET + BitUtil.SIZE_OF_INT;
     private static final int MESSAGES_RECEIVED_OFFSET = FRAMES_DECODED_OFFSET + BitUtil.SIZE_OF_LONG;
     private static final int FRAMES_ENCODED_OFFSET = MESSAGES_RECEIVED_OFFSET + BitUtil.SIZE_OF_LONG;
-    private static final int MESSAGES_SENT_OFFSET = FRAMES_ENCODED_OFFSET + BitUtil.SIZE_OF_LONG;
+    private static final int FILLER1_OFFSET = FRAMES_ENCODED_OFFSET + BitUtil.SIZE_OF_INT; // avoid 64b cache line
```

Comment:
Thank you for the patch @tonytamwk 
Could you change the field name from `FILLER1_OFFSET` to `CACHE_LINE_PADDING_OFFSET`?

Reply:
sure and done